app/views/propertyrepresentation/revokeAgentSummary.scala.html

```diff
@@ -18,28 +18,25 @@
 @import binders.pagination._
 @import binders.propertylinks._
 @import config.ApplicationConfig
+
+@this(mainLayout: views.html.mainLayout)
+
 @(
  action: AgentRevokeBulkAction,
  agentOrganisation: String
 )(implicit request: AuthenticatedRequest[_], messages: Messages, config: ApplicationConfig)
 
 @heading = @{ Messages("propertyRepresentation.agentRemoveSummary.heading") }
 
-@main_template(
-heading,
-contentHeaderText = Some(heading),
-contentHeaderHidden = true
+@mainLayout(
+ heading,
+ mainClass = Some("govuk-grid-column-full")
 ) {
 
-<div class="column-whole clearfix">
-
- <h1 class="heading-large">@messages("propertyRepresentation.agentRemoveSummary.heading")</h1>
-
- <p class="lede" id="mainBodyText">@messages("propertyRepresentation.agentRemoveSummary.removed", action.agentCode.toString, action.propertyLinkIds.size)</p>
-
+ <h1 class="govuk-heading-l">@messages("propertyRepresentation.agentRemoveSummary.heading")</h1>
 
- <a class="button" id="selectMorePropertiesButton" href="@agentAppointment.routes.AppointAgentController.selectAgentPropertiesSearchSort(PaginationParameters(), action.agentCode)" >@messages("propertyRepresentation.agentSummary.select.more")</a>
+ <p class="govuk-body" id="mainBodyText">@messages("propertyRepresentation.agentRemoveSummary.removed", action.agentCode.toString, action.propertyLinkIds.size)</p>
 
-</div>
+ <a class="govuk-button" id="selectMorePropertiesButton" href="@agentAppointment.routes.AppointAgentController.selectAgentPropertiesSearchSort(PaginationParameters(), action.agentCode)" >@messages("propertyRepresentation.agentSummary.select.more")</a>
```

Comment:
didn't the link look a bit smaller than usual in browser? I found that I had to put links into `govuk-body` p-tags for the links to have the right size.

Reply:
oh, i see it's "govuk-button" type of link... in that case shouldn't it be a govukButton component? 

something like this... ( from my own branch) 

`@govukButton(Button(content = Text(messages("invalidAccountType.register")), href = Some(routes.Register.show().url)))`

Reply:
So I am happy to go with either, I had this originally as a govUK button but amended it after a side discussion with Habeeb, it seemed to make sense to simply update the "button" to a "govUkButton" instead of introducing a new compoment, but also happy to revert it back to the GovUkButton - @habeeb-olayemi  what do you think?

Reply:
its a link that is styled as a button so I think what you have done is correct. I don't see a need to pull in the. govUkButton component

Reply:
I'll ask in plat UI channel, cuz I'd like to know. From design systems docs I can't see any use of a button-styled link.

Reply:
Okay cool, I'll hold fire on this for now

Reply:
![image](https://user-images.githubusercontent.com/1763911/118494614-4172b000-b71a-11eb-9483-7b7ba2a994f7.png)

Reply:
i think the govukButton component produces an `<a>` tag in the end, but the difference is that they can change the library to style the link differently and our button-styled links will automatically be updated if we pull in new version of design system library

Reply:
Updated back to button 👍🏻